test/api.js

```diff
@@ -4,8 +4,11 @@ var describe = require('mocha').describe
 
 var posthtml = require('../lib')
 
-var walk = require('../lib/api').walk
-var match = require('../lib/api').match
+var Api = require('../lib/api')
+var api = new Api()
```

Comment:
This should be instantiated in `PostHTML.prototype.process` instead to avoid the `.call` ?

Reply:
No, need only for extend `tree` because you call method the `tree` ```js tree.walk```

Reply:
major release... 
```
var walk = require('posthtml/lib/api').walk
VS
var Api  = require('posthtml/lib/api')
var api = new Api();
var walk = api.walk
```

Reply:
I do not want to, I'll fix it